CODEOWNERS

```diff
@@ -0,0 +1 @@
+@pankajastro @pankajkoti @tatiana
```

Comment:
```suggestion
@astronomer/astro-dagfactory-codeowners
```

Check instead if such an alias/group with adding those team members can be created & used here

Reply:
I just asked @phanikumv for us to have a single GH team for the OSS Integrations team, and then we could assign this both here and in the other repos. To maintain a team per project would be just additional maintenance overhead

Reply:
thanks @tatiana . @phanikumv could you please let me know once you create that team and the name of the team so I can add it here and in other repos, too?

Reply:
Added `oss-integrations` github team 

https://github.com/orgs/astronomer/teams/oss-integrations/members

Reply:
Thank you @phanikumv. Applied the change.